service/controller/v17/cloudconfig/master_template.go

```diff
@@ -274,6 +274,14 @@ func (e *MasterExtension) Units() ([]k8scloudconfig.UnitAsset, error) {
 			Enable:       false,
 			Command:      "start",
 		},
+		// Set bigger timeouts for NVME driver.
+		// Workaround for https://github.com/coreos/bugs/issues/2484
+		{
+			AssetContent: cloudconfig.NVMESetTimeoutsUnit,
+			Name:         "nvme-set-timeouts.service",
+			Enable:       true,
+			Command:      "start",
+		},
```

Comment:
Do we expect this to get rid of again at some point in the future? If yes, lets have a TODO issue so we do not forget about it.

Reply:
Yes, we do. Added. Thanks for pushing this!